src/main/resources/swagger/api-docs.yaml

```diff
@@ -2051,6 +2051,40 @@ paths:
             - email
             - profile
             - https://www.googleapis.com/auth/devstorage.full_control
+  /api/profile/billingAccounts:
+    get:
+      tags:
+        - Profile
+      operationId: billingAccounts
+      summary: List billing accounts for a user
+      responses:
+        200:
+          description: OK
+          produces:
+          - application/json
+          schema:
+            {
+              "type": "array",
+              "items": {
+                $ref: '#/definitions/BillingAccount'
+              }
+            }
```

Comment:
nitpick: I like using the simplified yaml here (writing without checking syntax, hope this works):

```
          schema:
            type: array
            items:
              $ref: '#/definitions/BillingAccount'
```